flink-streaming-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/common/RMQConnectionConfig.java

```diff
@@ -0,0 +1,455 @@
+	}
+
+	/**
+	 * Retrieve the requested heartbeat interval.
+	 * @return the initially requested heartbeat interval, in seconds; zero for none
+	 */
+	public int getRequestedHeartbeat() {
+		return requestedHeartbeat;
+	}
+
+	/**
+	 *
+	 * @return Connection Factory for RMQ
+	 * @throws Exception if Malformed URI has been passed
+     */
+	public ConnectionFactory getConnectionFactory() throws Exception {
+		ConnectionFactory factory = new ConnectionFactory();
+		if (this.uri != null && !this.uri.isEmpty()){
+			try {
+				factory.setUri(getUri());
+			}catch (Exception e){
+				LOG.error("Failed to parse uri {}", e.getMessage());
+				throw e;
+			}
+		} else {
+			factory.setHost(getHost());
+			factory.setPort(getPort());
+			factory.setVirtualHost(getVirtualHost());
+			factory.setUsername(getUsername());
+			factory.setPassword(getPassword());
+		}
+
+		factory.setAutomaticRecoveryEnabled(isAutomaticRecovery());
+		factory.setConnectionTimeout(getConnectionTimeout());
+		factory.setNetworkRecoveryInterval(getNetworkRecoveryInterval());
+		factory.setRequestedHeartbeat(getRequestedHeartbeat());
+		factory.setTopologyRecoveryEnabled(isTopologyRecovery());
+		factory.setRequestedChannelMax(getRequestedChannelMax());
+		factory.setRequestedFrameMax(getRequestedFrameMax());
+
+		return factory;
+	}
+
+	public static class Builder {
+		/** The default host */
+		public static final String DEFAULT_HOST = "localhost";
+
+		/** 'Use the default port' port */
+		public static final int USE_DEFAULT_PORT = -1;
+
+		/** Default virtual host */
+		public static final String DEFAULT_VHOST = "/";
+
+		/** Default user name */
+		public static final String DEFAULT_USER = "guest";
+
+		/** Default password */
+		public static final String DEFAULT_PASS = "guest";
+
+		/** The default connection timeout;
+		 *  zero means wait indefinitely */
+		public static final int DEFAULT_CONNECTION_TIMEOUT = 0;
+
+		/** Default maximum channel number;
+		 *  zero for unlimited */
+		public static final int DEFAULT_CHANNEL_MAX = 0;
+
+		/** Default maximum frame size;
+		 *  zero means no limit */
+		public static final int DEFAULT_FRAME_MAX = 0;
+
+		/** Default heart-beat interval;
+		 *  zero means no heart-beats */
+		public static final int DEFAULT_HEARTBEAT = 0;
+
+		private String host 		=  DEFAULT_HOST;
+		private int port 			=  USE_DEFAULT_PORT;
+		private String virtualHost  =  DEFAULT_VHOST;
+		private String username     =  DEFAULT_USER;
+		private String password     =  DEFAULT_PASS;
+
+		private int networkRecoveryInterval          = 5000;
+		private boolean automaticRecovery            = false;
+		private boolean topologyRecovery             = true;
+
+		private int connectionTimeout                = DEFAULT_CONNECTION_TIMEOUT;
+		private int requestedChannelMax              = DEFAULT_CHANNEL_MAX;
+		private int requestedFrameMax                = DEFAULT_FRAME_MAX;
+		private int requestedHeartbeat               = DEFAULT_HEARTBEAT;
```

Comment:
This formatting is not consistent with the rest of the Flink codebase. Can you just use a space before and after the equals sign?